nepal_compliance/overrides/salary_slip.py

```
@@ -0,0 +1,175 @@
+                )
+
+        if submitted:
+            try:
+                self.make_accrual_jv_entry(submitted)
+                self.email_salary_slip(submitted)
+                self.db_set({
+                    "salary_slips_submitted": 1,
+                    "status": "Submitted",
+                    "error_message": ""
+                })
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)